src/render/dom/html-visual-element.ts

```diff
@@ -40,6 +42,10 @@ export const htmlConfig: VisualElementConfig<
         vars: {},
     }),
 
+    sortNodePosition(a, b) {
+        return a.compareDocumentPosition(b) & 2 ? 1 : -1
```

Comment:
I wonder if a brief comment explaining this bitwise operation would be valuable? I read the spec and vaguely understand what's going on here but I will certainly forget 😅